src/buildercore/lifecycle.py

```diff
@@ -42,12 +42,12 @@ def push(lst, rec):
     if not lst or lst[-1] != rec:
         lst.append(rec)
 
-def restart(stackname):
+def restart(stackname, initial_states='pending|running|stopping|stopped'):
     """for each ec2 node in given stack, ensure ec2 node is stopped, then start it, then repeat with next node.
     rds is started if stopped (if *exists*) but otherwise not affected"""
-    start_rds_nodes(stackname)
+    # start_rds_nodes(stackname) # something a bit buggy here
```

Comment:
not sure if this should be kept disabled. I've seen the command fail here but don't have a stacktrace to confirm it.

... I'm not sure if anything else actually uses `restart` to be honest

Reply:
as far as I'm aware we don't use it in Jenkins, only separate `stop` and `start` commands in there